packages/web/src/components/gcds-notice/readme.md

```diff
@@ -0,0 +1,35 @@
+# gcds-alert
```

Comment:
Interesting, do you know why this says gcds-alert?

Reply:
I copied over the alert files as a base and just missed this one thing 😄